modules/ROOT/partials/integrations/react-tech-ref.adoc

```diff
@@ -163,10 +172,12 @@ include::partial$misc/get-an-api-key.adoc[]
 
 ===== Example: Using `apiKey`
 
-```jsx+++<Editor apiKey="your-api-key">++++++</Editor>+++
-
-```
+[source, jsx]
+----
+  <Editor apiKey="your-api-key"></Editor>
+----
 
+[[cloudchannel]]
 ==== `cloudChannel`
 
 Changes the {productname} build used for the editor to either a specific version or a channel indicating a stability level.
```

Comment:
The example below needs `subs="attributes+"` added.

Reply:
done